pkg/ccl/streamingccl/streamingest/replication_stream_e2e_test.go

```diff
@@ -929,7 +929,8 @@ func TestProtectedTimestampManagement(t *testing.T) {
 	defer leaktest.AfterTest(t)()
 	defer log.Scope(t).Close(t)
 
-	skip.UnderRace(t, "slow test") // takes >1mn under race.
+	skip.UnderRace(t, "slow test")    // takes >1mn under race.
+	skip.UnderStress(t, "flaky test") // flaky under stress testing.
```

Comment:
nit: i think we can remove the stress skip now that https://github.com/cockroachdb/cockroach/pull/125534 has landed